core/services/keeper/orm_test.go

```diff
@@ -136,20 +136,27 @@ func TestKeeperDB_BatchDeleteUpkeepsForJob(t *testing.T) {
 
 	registry, job := cltest.MustInsertKeeperRegistry(t, db, orm, ethKeyStore, 0, 1, 20)
 
+	var upkeepIDs []utils.Big
+	var expectedUpkeepID int64
 	for i := int64(0); i < 3; i++ {
-		cltest.MustInsertUpkeepForRegistry(t, db, config, registry)
+		upkeep := cltest.MustInsertUpkeepForRegistry(t, db, config, registry)
+		if i == 1 {
+			expectedUpkeepID = upkeep.UpkeepID.Int64()
+		} else {
```

Comment:
nit. Assign this outside the loop? i.e. 

expectedUpkeepID := cltest.MustInsertUpkeepForRegistry(t, db, config, registry) outside the for loop